apps/marketplace/components/BuyerSpecialist/BuyerSpecialistAdditionalInformationStage.js

```diff
@@ -215,23 +250,30 @@ export class BuyerSpecialistRequirementsStage extends Component {
 
 BuyerSpecialistRequirementsStage.defaultProps = {
   onSubmit: () => {},
-  onSubmitFailed: () => {}
+  onSubmitFailed: () => {},
+  lockoutPeriod: {
+    startDate: null,
+    endDate: null
+  }
 }
 
 BuyerSpecialistRequirementsStage.propTypes = {
   model: PropTypes.string.isRequired,
   saveModel: PropTypes.func.isRequired,
   formButtons: PropTypes.node.isRequired,
   onSubmit: PropTypes.func,
-  onSubmitFailed: PropTypes.func
+  onSubmitFailed: PropTypes.func,
+  lockoutPeriod: PropTypes.object
 }
 
 const mapStateToProps = (state, props) => ({
-  ...formProps(state, props.model)
+  ...formProps(state, props.model),
+  lockoutPeriod: state.brief.lockoutPeriod
 })
 
 const mapDispatchToProps = (dispatch, props) => ({
-  setDate: date => dispatch(actions.change(`${props.model}.closedAt`, date))
+  setDate: date => dispatch(actions.change(`${props.model}.closedAt`, date)),
+  loadInitialData: briefId => dispatch(loadPublicBrief(briefId))
```

Comment:
The other additional information stages don't appear to have this.  Is it required or can it be removed?

Reply:
No it doesn't seem necessary. I can't find any references which seem dependent on the code, and when I removed it, functionality seems perfectly fine in my testing, without any errors displayed in the console.